src/calendar/view/CalendarViewModel.ts

```diff
@@ -381,11 +382,13 @@ export class CalendarViewModel implements EventDragHandlerCallbacks {
 			return
 		}
 		const eventListId = getListId(event)
-		const eventMonth = getMonthRange(getEventStart(event, this.timeZone), this.timeZone)
-		if (isSameId(calendarInfo.groupRoot.shortEvents, eventListId) && this._loadedMonths.has(eventMonth.start)) {
-			// If the month is not loaded, we don't want to put it into events.
-			// We will put it there when we load the month
-			this._addDaysForEvent(event, eventMonth)
+		if (isSameId(calendarInfo.groupRoot.shortEvents, eventListId)) {
```

Comment:
this is only called from `entityEventsReceived()` but the issues are about just opening the calendar? Am I wrong?

Reply:
as far as I know and could reproduce, this is purely an issue when we receive the event through an entity update, they appear correctly once you reload the page